src/config.js

```diff
@@ -1,5 +1,20 @@
-export const dataPath = './api/main';
-export const fullDataPath = `/public${dataPath.substr(1)}`;
+/**
+ * Determine where to load data from
+ * @param {string} type Data type
+ * @param {string=} id
+ */
+export const getUrl = (type, id) => {
```

Comment:
We should probably avoid functions in config, ideally break out the constants here and put the function elsewhere I think, especially since it isn't a pure function.

Reply:
okay fair enough, I'll move it to `utils/index.js` I guess, and split out the string parts into an object and put them in config.